packages/component-login/client/index.js

```diff
@@ -67,6 +67,7 @@ class LoginPage extends React.Component {
         LoginPage.setToken(null)
         // TODO expose this error once we have a UI to do so
         this.props.history.push('/login', { error: err.message })
+        // eslint-disable-next-line no-console
```

Comment:
why not remove the console.log?
if you need to log anything important, use logger

Reply:
will need to change this in a separate PR and add the login test configs.